src/Microsoft.ComponentDetection.Detectors/poetry/Contracts/PoetryPackage.cs

```diff
@@ -15,5 +16,9 @@ public class PoetryPackage
 
         [SuppressMessage("StyleCop.CSharp.NamingRules", "SA1300:ElementMustBeginWithUpperCaseLetter", Justification = "Deserialization contract. Casing cannot be overwritten.")]
         public PoetrySource source { get; set; }
+
+        public TomlTable dependencies { get; set; }
```

Comment:
```suggestion

        [DataMember(Name = "dependencies")]
        public TomlTable Dependencies { get; set; }
```

Can you add a `DataMember` attribute to this, and all other properties in this class? This means you can also get rid of the `SuppressMessage` attribute.

Can you also add a `[DataContract]` attribute to the class please?